index.bs

```diff
@@ -2953,6 +2953,287 @@ value and terminate the operation.
 
 </div>
 
+### Signal Credential Changes to the Authenticator - PublicKeyCredential's [=signal methods=] ### {#sctn-signal-methods}
+
+<xmp class="idl">
+    partial interface PublicKeyCredential {
+        static Promise<undefined> signalUnknownCredentialId(UnknownCredentialIdOptions options);
+        static Promise<undefined> signalAllAcceptedCredentialIds(AllAcceptedCredentialIdsOptions options);
```

Comment:
What do you think about dropping off `Id` from just the method names? It makes the names a _little_ shorter which seems doable because of an implication that an ID will be specified within the options (e.g. "because how else would you identify the credential?").

So `signalAllAcceptedCredentialIds` becomes `signalAllAcceptedCredentials`:

```
await PublicKeyCredential.signalAllAcceptedCredentials({
  rpId: "example.com",
  userId: "M2YPl-KGnA8", // same as user.id at creation time
  allAcceptedCredentalIds: [
     // IDs of all accepted credentials
    "vI0qOggiE3OT01ZRWBYz5l4MEgU0c7PmAA",
    "Bq43BPs"
  ]
});
```

...and `signalUnknownCredentialId` becomes `signalUnknownCredential`:

```
await PublicKeyCredential.signalUnknownCredential({
  rpId: "example.com",
  credentialId: "vI0qOggiE3OT01ZRWBYz5l4MEgU0c7PmAA"  // b64-url
});
```

I think in aggregate it sets a precedent for slightly more human-looking API names especially with `signalCurrentUserDetails` in the mix.

Note that I didn't change any of the names of the options. I'm only proposing alternative names for the methods.

Reply:
Done.